ext/preprocess/FileSystemGlobbingInternalization.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
```

Comment:
Can we put this code in a project so that we get an editor experience around it?